force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -182,15 +234,11 @@ public with sharing class FlowControlLoopsCollections {
      * @return A List of integers containing the numbers 1, 2, and 3.
      */
     public static List<Integer> createAndPopulateList() {
-        // Create a new list of integers
-
-        // Add the integers 1, 2, and 3 to the list
-        // add 1
-        // add 2
-        // add 3
-
-        // Return the populated list
-        return null; // Replace null with the variable you used to store the result
+        List<Integer> myList = new List<Integer>();
+        for(Integer i = 1; i< 4; i++) {
+            myList.add(i);
+        }
+        return myList; 
```

Comment:
Good job using a loop. Senior dev tip. If you know your values ahead of time. It is easier to use the List constructor. 

```
 public static List<Integer> createAndPopulateList() {
      return new List<Integer>{1, 2, 3};
    }
 ```